models/gif_cond_exp_multisynapse.cpp

```diff
@@ -140,28 +140,31 @@ nest::gif_cond_exp_multisynapse::Parameters_::Parameters_()
   , q_sfa_()           // mV
   , tau_syn_()         // ms
   , I_e_( 0.0 )        // pA
-  , receptor_types_()
   , num_of_receptors_( 0 )
   , E_ex_( 0.0 )   // mV
   , E_in_( -85.0 ) // mV
   , has_connections_( false )
+  , gsl_error_tol( 1e-3 )
 {
 }
 
 nest::gif_cond_exp_multisynapse::State_::State_( const Parameters_& p )
-  : y_( NUMBER_OF_FIXED_STATES_ELEMENTS, 0.0 )
-  , y0_( 0.0 )
+  : y_( NULL )
+  , size_neuron_state_( 0 )
+  , I_stim_( 0.0 )
   , sfa_( 0.0 )
   , stc_( 0.0 )
   , sfa_elems_()
   , stc_elems_()
   , r_ref_( 0 )
 {
+  y_ = ( double* ) malloc( NUMBER_OF_FIXED_STATES_ELEMENTS * sizeof( double ) );
```

Comment:
I have added a  new method (`neuron_state_memory_allocate`) to do this job. A also replaced the old fashion casting with `reinterpret_cast`.